functions/airtable-connector.js

```diff
@@ -70,7 +73,7 @@ function toDataJson(entries) {
   };
 }
 
-function loadMakerData(admin, req, res) {
+function loadFromAirtable(static_data, admin, req, res) {
   const base = new Airtable({apiKey: 'key7sP7fMv2drjUnJ'}).base('appmMuprpuCFw4wCd');
```

Comment:
config vars -> google cloud env? Should we also create a new key?

Reply:
moved. Will generate new key later.